### DISCUSSION TITLE
Installed Pybullet From Pip, Still Printing Build time

### DISCUSSION THREAD

**Discussion by fishbotics** [bulletphysics/bullet3#3731] [General]
Hi,

I installed Pybullet from pip as instructed in the README. Whenever I import it, it prints the build time, which should only be happening in debug mode. Is the binary distributed with pip the debug build? Is there some flag I can set? 

It wasn't really bothering me until I started doing multithreading and now it's printing all over the place, which is pretty distracting from my actual logs.

Thanks!

**Comment by erwincoumans** [comment]
Let me have a look, should be a quick fix (will add a method to query for build time string). If it us urgent, edit Bullet/examples/pybullet/pybullet.c at the very end, comment-out the printf. Then use python3 setup.py install --user in the root of Bullet.
